src/broadcast/metadatafileworker.cpp

```diff
@@ -0,0 +1,35 @@
+
+#include "metadatafileworker.h"
+
+MetadataFileWorker::MetadataFileWorker(const QString &filePath)
+        :  m_file(filePath) {
+
+}
+
+void MetadataFileWorker::slotDeleteFile() {
+    m_file.remove();
+}
+
+void MetadataFileWorker::slotOpenFile() {
+    m_file.open(QIODevice::ReadWrite |
```

Comment:
Where do we close the file?

Reply:
Oh, you wanted to open, write and close every time right? I forgot

Reply:
I was unsure if this is required. In this case I just want a final close.

Reply:
I changed it in the new commit, I open, write and close.